### PR TITLE
Fab rename symbols

### PR DESCRIPTION
**Pull request by taquitos**
Rename all symbols so there are no collisions when other apps include JLObjectMapping

### CODE REVIEW

File: JLJSONMapping/Category/NSError+JLJSONMapping.m [hunk 1]
```diff
@@ -8,24 +8,24 @@
 
 #import "NSError+JLJSONMapping.h"
 
-NSString * const kObjectMappingDomain = @"com.mydoghatestechnology.objectmapping";
-NSString * const kObjectMappingDescriptionKey = @"JLObjectMappingDescriptionKey";
-NSString * const kObjectMappingFailureReasonKey = @"JLObjectMappingDetailedFailureReasonKey";
```
Comment (JaviSoto) [inline JLJSONMapping/Category/NSError+JLJSONMapping.m:13]: my dog hates technology? hahahahhaha

File: JLJSONMapping/Category/NSError+JLJSONMapping.m [hunk 2]
```diff
@@ -8,24 +8,24 @@
...
+NSString * const kFABObjectMappingDomain = @"com.mydoghatestechnology.objectmapping";
```
Comment (JaviSoto) [inline JLJSONMapping/Category/NSError+JLJSONMapping.m]: Do we wanna change the string value of this too? That way they'll really be different domains.

**Comment by JaviSoto** [comment]
It looks great :)

File: JLJSONMapping/Category/NSError+JLJSONMapping.m (hunk 1 above)
Comment (taquitos) [inline JLJSONMapping/Category/NSError+JLJSONMapping.m:13]: yeah, my old dog used to eat my electronics so I use that domain now.
dog destruction

File: JLJSONMapping/Category/NSError+JLJSONMapping.m (hunk 2 above)
Comment (taquitos) [inline JLJSONMapping/Category/NSError+JLJSONMapping.m]: Good call

File: JLJSONMapping/Category/NSError+JLJSONMapping.m (hunk 1 above)
Comment (JaviSoto) [inline JLJSONMapping/Category/NSError+JLJSONMapping.m:13]: tumblr_inline_nniy772az11tsljqh_500

**Pull request merged**